components/inputs/test/input-number.vdiff.js

```diff
@@ -25,6 +26,14 @@ describe('d2l-input-number', () => {
 				</d2l-input-number>
 			`
 		},
+		{
+			name: 'percentage-input-width-after-slot',
+			template: html`
+				<d2l-input-number label="Help Text" input-width="50%">
+					<d2l-button-icon icon="tier1:help" text="help" slot="after"></d2l-button-icon>
+				</d2l-input-number>
```

Comment:
I noticed in the number input demo that the example for "after slot" is using a `<d2l-button-icon>` just like this test, but it's not rendering because that component isn't imported. Could you fix that as part of this PR?